src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -91,7 +91,7 @@ private static List<CompletionResult> CompleteCommand(CompletionContext context,
 
             List<CompletionResult> commandResults = null;
 
-            if (commandName.IndexOfAny(Utils.Separators.DirectoryOrDrive) == -1)
+            if (commandName.AsSpan().IndexOfAny('\\', '/', ':') == -1)
```

Comment:
I still think `Separators.DirectoryOrDrive` is more readable than `'\\', '/', ':'` -- the code becomes self-explainable.

Reply:
Every usage already has comments in code and the comments are more informative.

Reply:
I don't see comment here and a few other places where this is used.

Reply:
```c#
            if (commandName.AsSpan().IndexOfAny('\\', '/', ':') == -1)
            {
                // The name to complete is neither module qualified nor is it a relative/rooted file path.
```
I wonder if we need better comment :-)

Reply:
I missed that comment, but `DirectoryOrDrive` is still a good supplement to that comment.

I think we should keep some of the members in `Separators` as is, and `DirectoryOrDrive` is one of them because:
1. it's used at 4 places, so using a static array to capture them and reuse makes sense.
2. I don't think this change will bring any measurable performance difference, so we should not make changes just for the sake of using Span APIs.

For those APIs where an overload is added that takes a char directly, it makes sense to just use that API, but for instances like this one, I don't think it's really worth the changes.